modelon/impact/client/sal/service.py

```diff
@@ -82,14 +82,18 @@ def wrapped(*args: Any, **kwargs: Any) -> Callable:
             self.external_result, retry_with_login_decorator
         )
 
+    def is_jupyterhub_url(self) -> bool:
+        url = (self._base_uri / "api/").resolve()
```

Comment:
I think this is preferable. Otherwise we don't really know if `/api` respose comes from impact or jupyterhub (so far versions are different but that is not reliable). So it should be either 200 -> jupyterhub/true or 404 -> not jupyterhub/false -> othewize comm exception is fine IMO as the answer would be unexpected.